entities/dashboard/widget.json

```diff
@@ -0,0 +1,128 @@
+{
+    "type": "object",
+    "properties": {
+        "uid": {
+            "type": "string",
+            "description": "Unique identifier of the dashboardWidget object"
+        },
+        "app_code_name": {
+            "type": "string",
+            "description": "The code name of the app that owns the widget"
+        },
+        "display_name": {
+            "type": "object",
+            "description": "The widget display name in different languages"
+        },
+        "permissions": {
+            "type": "string",
+            "description": "The permissions list required to be able to use this widget, seperated bu comma"
+        },
+        "feature_flags": {
+            "type": "string",
+            "description": "The feature flags list required to be able to use this widget, seperated by comma"
```

Comment:
This should be an array. Please do not use a comma-separated string unless explicitly approved by Eldad and Niki. This practice is a short cut that causes us a huge headache when we try to do reporting on a feature or understand setup configuration.

Alternatively, we should only support one feature flag per widget. This is usually a single package and I don't see a reason why we should treat it as a list. If this is ok with you, then change the description accordingly:

"The feature flag required for the widget to be shown to a staff member. Typically, a package feature flag that indicates the existence of a feature in the business account subscription plan, for instance, "bizai_platform".